src/pybind/mgr/dashboard/health.html

```diff
@@ -366,6 +366,7 @@
                             <th>PG status</th>
                             <th>Usage</th>
                             <th>Activity</th>
```

Comment:
"Activity" is now a bit of an awkward column title because it doesn't make it clear that we're talking about bytes as opposed to ops... I don't immediately have a replacement suggestion though